Updated AI-conformance annual report (#8842)
kubernetes/community: wg-ai-conformance/annual-report-2025.md

   *   **Launch of the Kubernetes AI Conformance Program:** The group launched the official AI conformance program. This included the creation of the `cncf/k8s-ai-conformance` repository to host the program's resources and certification process, and an announcement at KubeCon + CloudNativeCon North America 2025.

   *   **Release of Conformance Checklists:** The WG defined and released conformance checklists for multiple Kubernetes versions, providing a clear baseline for AI-ready clusters:
        *   [`AIConformance-1.33.yaml`](https://github.com/cncf/k8s-ai-conformance/blob/main/docs/AIConformance-1.33.yaml)
        *   [`AIConformance-1.34.yaml`](https://github.com/cncf/k8s-ai-conformance/blob/main/docs/AIConformance-1.34.yaml)

>         *   [`AI Conformance-1.35.yaml`](https://github.com/cncf/k8s-ai-conformance/blob/main/docs/AIConformance-1.35.yaml)

Reviewer: nit
```suggestion
        *   [`AIConformance-1.35.yaml`](https://github.com/cncf/k8s-ai-conformance/blob/main/docs/AIConformance-1.35.yaml)
```

Author: Oops, fixed

   *   **Established Requirement & Test Processes:**
        *   Transitioned requirement tracking from documents to a structured GitHub Project: [WG AI Conformance Requirements](https://github.com/orgs/kubernetes-sigs/projects/114).
        *   Defined the [Kubernetes AI Conformance Requirement (KAR)](https://github.com/kubernetes-sigs/wg-ai-conformance/tree/main/kars/NNNN-kar-template) process, mirroring the KEP process, to manage the lifecycle of requirements (SHOULD/MUST).

>    *   **Graduation to Subproject:** Following the successful launch of the conformance program, the Working Group officially transitioned into a permanent subproject under SIG Architecture in February 2026 to ensure long-term maintenance of the program.

Reviewer 2: Since this is annual report for 2025, should we say

```suggestion
   *   **Graduation to Subproject:** Following the successful launch of the conformance program, the Working Group will transition to a permanent subproject under SIG Architecture in 2026 to ensure long-term maintenance of the program.
```

Author: Fixed
